### DISCUSSION TITLE
浦发银行开源实践：开源项目活跃度模型构建及实证

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#45] [案例分享 & 最佳实践]
分享一篇来自浦发银行的杨欣捷、田蜜、江一鸣几位一线专业人士的开源实践，很有启发~

摘要：随着开源技术不断涌现，开源作为一种治理模式、经济模式和企业生态，变得越来越重要。关注开源项目的活跃程度 ，分析活跃度的变化情况 ，在一定层面上能够反映项目发展状态。通过梳理现有的开源项目活跃度模型，发现：第一 ，现有模型没有考虑项目类别，不能有效反映镜像类项目活跃度； 第二，现有模型没有考虑活跃度趋势，不能有效反映阶段性活跃项目的整体活跃度 。在此基础上构建和完善了活跃度模型，并以浦发银行开源治理平台为例，对模型进行了验证，为企业在开源项目的选择上提供一定的参考价值。

[开源项目活跃度模型构建及实证_杨欣捷.pdf](https://github.com/kaiyuanshe/ONES/files/8260376/_.pdf)

**Comment by wangleo61** [comment]
很有幸和杨欣捷老师、江一鸣老师合作过，两位对于开源的认识非常具有前瞻性，同时也是深耕于开源的技术领域的研究，非常敬佩
